src/Run.hs

```diff
@@ -69,8 +71,14 @@ run dir startupFile args = do
       go = do
         status <- withSession startupFile args $ \ session -> do
           let
-            triggerAction = saveOutput (trigger session)
-            triggerAllAction = saveOutput (triggerAll session)
+            mkTriggerAction theTrigger = do
+              result@(success, _) <- theTrigger session
+              saveOutput (return result)
+              if success
+                then void $ senseiHooksOnSuccess $ sessionHooks session
+                else void $ senseiHooksOnFailure $ sessionHooks session
```

Comment:
Do we want to notify the user if running the command fails? If yes, see my previous comment.

Reply:
done